docs/install.md

```diff
@@ -24,10 +24,18 @@ The codespace includes Mailhog, which will trap any emails sent by MyRadio. To s
 click the Local Address next to port 8025 in the Ports panel.
 
 ## Docker Install
+
 If you have Docker on your system, use Docker Compose to set up an environment.
-Simply run `docker compose up -d`, and visit "https://localhost:4443/myradio/".
+Simply run `docker compose up -d` and visit "https://localhost:4443/myradio/".
+
+If you encounter an error with autoload.php:
+find the id of your myradio container using `docker dontainer ls`
+enter a bash session with `docker exec -it [myradioid] bash`
+then in this session run `composer install`
+finally exit the session by running `exit`
```

Comment:
This should probably be `docker compose exec myradio bash`, rather than looking up the container ID

Reply:
I've had issue getting `docker compose exec myradio bash` to work even from a fresh docker, I tell people to use ids as it's more likely to work than not. Rather than confusing someone that might have a different name for their myradio instance.